src/components/SelectButtonGroup/SelectButton/SelectButton.scss

```diff
@@ -1,49 +1,96 @@
 .SelectButton {
-    box-shadow: inset 0 -3px 0 var(--color-neutral-25);
-    height: 38px;
     box-sizing: border-box;
-    padding: var(--spacing-3x) var(--spacing-4x);
-    border: 1px solid var(--color-neutral-40);
-    border-right: none;
+    border: var(--line-normal) solid transparent;
+    padding: var(--spacing-2x) var(--spacing-4x);
+    line-height: var(--line-height-normal);
     white-space: nowrap;
     text-overflow: ellipsis;
     text-align: center;
     overflow: hidden;
     cursor: pointer;
+    outline: none;
+    color: var(--color-neutral-75);
+    font-weight: var(--font-weight-bold);
 
     &:first-child {
         border-top-left-radius: var(--border-radius);
         border-bottom-left-radius: var(--border-radius);
     }
     &:last-child {
-        border-right: 1px solid var(--color-neutral-40);
         border-top-right-radius: var(--border-radius);
         border-bottom-right-radius: var(--border-radius);
     }
 
-    &:hover,
-    &:focus {
-        background: var(--color-neutral-10);
-        outline: none;
+    &:hover {
+        background: var(--color-neutral-25);
+    }
+
+    + .SelectButton {
+        box-shadow: -1px 0 0 0 var(--color-background);
+    }
+
+    &[disabled] {
+        cursor: not-allowed;
     }
 
     &--isSelected.SelectButton--selectedContext_neutral {
```

Comment:
would it make it easier to read this if this and the next 2 rules would be combined?
```
&--isSelected.SelectButton--selectedContext {
  &_neutral {...}
  &_good {...}
  &_brand {...}
}
```

Reply:
We could indeed do that for `good` and `brand` but not including `neutral` because its styles (`border-color` and `background-color`) are different. Maybe at this point we should just make it support any context out of the box instead of a particular subset.

Reply:
sure, if you think that's easy. we haven't implemented the other contexts before, because we had no design for them

Reply:
you can still scope the rules though as above, they can all have their own rules